src/backend/metal/src/map.rs

```diff
@@ -237,16 +237,13 @@ pub fn map_format(format: Format, is_target: bool) -> Option<MTLPixelFormat> {
             Int   => RGBA8Sint,
             Uint  => RGBA8Uint,
             Inorm => RGBA8Snorm,
-            Unorm => if is_target {
-                         BGRA8Unorm
```

Comment:
IIRC it's as mentioned in the PR description, the backbuffer requires BGRA and gfx_app used RGBA everywhere and I did the conversion (stupidly) in that conversion function (`map_format`). This looks much cleaner (and most importantly more functional!)